### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by srishtigautam21**
Made the card component
**Folders/Files to check:**
1.Card
2.Context
3.db->product.js
4.ProductPage.jsx

### CODE REVIEW

File: src/component/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -42,20 +42,23 @@ const Navbar = () => {
...
+          <li className='list-style icons-alignment'>
+            <Link className='link ecom-link-color' to='/products'>
               <HeartIcon className='nav-icons' />
               <p>Wishlist</p>
             </Link>
           </li>
-          <li class='list-style icons-alignment'>
-            <a class='link ecom-link-color' href='/Pages/LoginPage/login.html'>
+          <li className='list-style icons-alignment'>
+            <a
```
Comment (santhoshpemmaka) [inline src/component/Navbar/Navbar.jsx]: Better to use a Link tag instead of a tag. a tag refresh full page.

**Comment by santhoshpemmaka** [comment]
Code looking good and clean structure 💯 . Good to merge 👍 .

**Review by santhoshpemmaka — APPROVED** [review]

File: src/component/Navbar/Navbar.jsx (hunk 1 above)
Comment (srishtigautam21) [reply src/component/Navbar/Navbar.jsx]: Thanks santhosh i had forgot to change it here

**Review by purvasharmadev — APPROVED** [review]
everything is looking nice, good to go.
good job with folder structuring.

**Pull request merged**